### ISSUE TITLE
Using stryker-cli on macOS

### ISSUE THREAD

**Issue by korthout** [stryker-mutator/stryker-cli#23]
So having helped develop, I didn't encounter any problems when running `stryker init` locally. However, today I installed `styker-cli` globally using npm and encountered the following issue.

```
> stryker init
env: node\r: No such file or directory
```

**Comment by korthout** [comment]
After some googling I discovered that this has to do with a mismatch in linebreak characters used by windows/dos-style `\r\n` and unix-style `\n`. 

I found the following work around at [StackOverflow](https://stackoverflow.com/a/31041038).

**Note** that this solution uses [Homebrew](https://brew.sh) to install the `dos2unix` package.

```
brew install dos2unix
find ~/.node/lib/node_modules/stryker-cli/ -name "*.js" | xargs sudo dos2unix
find ~/.node/lib/node_modules/stryker-cli/bin/stryker-cli | xargs sudo dos2unix
```

After executing this, everything seemed to work fine.

```
> stryker init
Stryker is currently not installed.
? Do you want to automatically install Stryker?
```

**Comment by korthout** [comment]
@nicojs @simondel @avassem85 I think users may avoid the stryker-cli when encountering this behaviour, so I think we should see if we can solve this issue in general, instead of having to use the above workaround.

**Comment by nicojs** [comment]
### TL;DR;

We're missing a `.gitattributes` file. We do, however, have a `.gitatrributes` file :unamused: (facepalm) . We need to rename the .gitattributes.

### Long version

Git is more than a distribute source control utility. It's also a line ending management tool (i don't like that part, but it is). When installing git on windows, you get the questions on what to do with line endings. I (the person who did the release) chose the option "checkout windows style, check in unix style". This means that my local `bin/stryker-cli` file is different from yours. Mine has CR+LF line endings. 

For me, everything works locally. So i did a release from my pc. However, npm is not a line ending management tool. It does not honor my git settings and it created a release with the CR+LF line endings.

My global settings for windows style line endings can be overruled per repo using the [.gitattributes](https://help.github.com/articles/dealing-with-line-endings/#per-repository-settings) file. After we've created that file, problems should be solved forever, as everyone is forced to use unix style `lf` line endings.

**Comment by nicojs** [comment]
![Lies](https://user-images.githubusercontent.com/1828233/27230925-2db8dc00-52b1-11e7-9a50-3a7d1ace4d0c.PNG)

No @avassem85 . You did not! Lies! Lies! Lies!

**Comment by korthout** [comment]
I have just verified. Problem is solved by PR #24.
